pkg/sdk/tags.go

```diff
@@ -15,6 +15,8 @@ type Tags interface {
 	Undrop(ctx context.Context, request *UndropTagRequest) error
 	Set(ctx context.Context, request *SetTagRequest) error
```

Comment:
I disagree because in `SetOnCurrentAccount` we don't use any identifier. We're modifying the current account like `ALTER ACCOUNT`, where this one uses `ALTER <type> <id>` which is different.